From gadiyar@ti.com  Fri Nov 20 12:46:28 2009
From: Mike Frysinger <vapier@gentoo.org>
Date: Mon, 16 Nov 2009 16:19:29 +0530
Subject: USB: musb: fix printf warning in debug code
To: "linux-usb@vger.kernel.org; Greg KH" <greg@kroah.com>
Cc: Mike Frysinger <vapier@gentoo.org>, Anand Gadiyar <gadiyar@ti.com>
Message-ID: <1258368569-4323-11-git-send-email-gadiyar@ti.com>


From: Mike Frysinger <vapier@gentoo.org>

The debug code in the DMA ISR uses a %d for a size_t when it should be
using %zu.  Otherwise gcc whines with:

drivers/usb/musb/musbhsdma.c: In function 'dma_controller_irq':
drivers/usb/musb/musbhsdma.c:288: warning: format '%d' expects type 'int',
	but argument 7 has type 'size_t'

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
Signed-off-by: Anand Gadiyar <gadiyar@ti.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/musb/musbhsdma.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/musb/musbhsdma.c
+++ b/drivers/usb/musb/musbhsdma.c
@@ -285,7 +285,7 @@ static irqreturn_t dma_controller_irq(in
 				channel->actual_len = addr
 					- musb_channel->start_addr;
 
-				DBG(2, "ch %p, 0x%x -> 0x%x (%d / %d) %s\n",
+				DBG(2, "ch %p, 0x%x -> 0x%x (%zu / %d) %s\n",
 					channel, musb_channel->start_addr,
 					addr, channel->actual_len,
 					musb_channel->len,
